bin/Bin.re

```diff
@@ -1,3 +1,3 @@
 open Ppxlib;
 
-let _ = Driver.standalone();
+let _ = Driver.run_as_ppx_rewriter();
```

Comment:
Using `standalone` here allows for more options at runtime, but requires a wrapper around the ppx that passes `--as-ppx` argument before the arguments that the compiler used.